qa/suites/orch/rook/smoke/3-final.yaml

```diff
@@ -6,3 +6,4 @@ tasks:
       - ceph orch ls
       - ceph orch host ls
       - ceph orch device ls
+      - ceph orch apply rgw foo
```

Comment:
we need to fix the rook teardown somehow..
```
2021-09-19T10:36:03.666 INFO:tasks.rook.operator.smithi016.stdout:2021-09-19 10:36:03.669485 I | ceph-object-controller: created object store "foo" in namespace "rook-ceph"
2021-09-19T10:36:04.365 INFO:tasks.rook.operator.smithi016.stdout:2021-09-19 10:36:04.368811 I | ceph-cluster-controller: CephCluster "rook-ceph/rook-ceph" will not be deleted until all dependents are removed: CephObjectStores: [foo]
2021-09-19T10:36:04.378 INFO:tasks.rook.operator.smithi016.stdout:2021-09-19 10:36:04.381882 E | ceph-cluster-controller: failed to reconcile CephCluster "rook-ceph/rook-ceph". CephCluster "rook-ceph/rook-ceph" will not be deleted until all dependents are removed: CephObjectStores: [foo]
2021-09-19T10:36:14.599 INFO:tasks.rook.operator.smithi016.stdout:2021-09-19 10:36:14.601708 I | ceph-cluster-controller: CephCluster "rook-ceph/rook-ceph" will not be deleted until all dependents are removed: CephObjectStores: [foo]
2021-09-19T10:36:14.615 INFO:tasks.rook.operator.smithi016.stdout:2021-09-19 10:36:14.619152 E | ceph-cluster-controller: failed to reconcile CephCluster "rook-ceph/rook-ceph". CephCluster "rook-ceph/rook-ceph" will not be deleted until all dependents are removed: CephObjectStores: [foo]
```/a/sage-2021-09-19_00:55:24-rados-wip-sage-testing-2021-09-18-1754-distro-basic-smithi/6396859